files/akamai-cli-config

```diff
@@ -6,3 +6,4 @@ config-version        = 1.1
 stats-version         = 1.1
 install-in-path       = 
 last-ping             = 2050-01-01T00:00:00Z
+cache-path            = /cli/.akamai-cli/cache
```

Comment:
Small gripe here that we have a hardcoded value for $AKAMAI_CLI_HOME, and IF we were to change it's value in the dockerfiles we might forget to do so here.
Easiest might be to add a small comment in the cli.Dockerfile where we assign the variable, as a reminder to change the value in akamai-cli-config as well. Possibly also a small comment on why we have overridden this to be /cli if you think it useful.

Reply:
Good idea - updated already!